src/app/frontend/replicationcontrollerdetail/replicationcontrollerdetail.html

```diff
@@ -33,241 +30,220 @@ <h1 flex class="md-title kd-replicationcontrollerdetail-app-name">
     </kd-replication-controller-info>
   </div>
   <div flex>
-  <div class="kd-replicationcontrollerdetail-sidebar-header-wrapper kd-replicationcontrollerdetail-sidebar-item"
-       flex="nogrow" ng-show="!ctrl.isSidebarVisible()">
-    <div class="kd-replicationcontrollerdetail-sidebar-header" layout-align="start center"
-         layout="row">
-      <md-button class="md-icon-button" ui-sref="replicationcontrollers">
-        <md-icon md-font-library="material-icons">arrow_back</md-icon>
-      </md-button>
-      <h1 flex class="md-title kd-replicationcontrollerdetail-app-name">
+    <div class="kd-replicationcontrollerdetail-sidebar-header-wrapper kd-replicationcontrollerdetail-sidebar-item"
+      flex="nogrow" ng-show="!ctrl.isSidebarVisible()">
+      <div class="kd-replicationcontrollerdetail-sidebar-header" layout-align="start center" layout="row">
+        <md-button class="md-icon-button" ui-sref="replicationcontrollers">
+          <md-icon md-font-library="material-icons">arrow_back</md-icon>
+        </md-button>
+        <h1 flex class="md-title kd-replicationcontrollerdetail-app-name">
         {{::ctrl.replicationControllerDetail.name}}
       </h1>
+      </div>
```

Comment:
`<div>` content should be indented. `{{::ctrl.replicationControllerDetail.name}}` and `</h1>` are not. It's wrong.